uint8 where   # Colocar o nÃºmero da figura
---

